pkg/numaresourcesscheduler/manifests/yaml/networkpolicy.apiserver.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: scheduler-egress-to-api-server
+spec:
+  podSelector:
+    matchLabels:
+      app: secondary-scheduler 
```

Comment:
this is namespaced, right?

Reply:
the network policies are supposed to be namespaced, but they are copied from the kustomize directory which adds the namespace on the fly.

I think we'll have to add them later on d/s

Reply:
it's fine, I was just asking if this resource is per-cluster or namespaced. If it's namespaced than we can easily fix the namespace in many ways and my concerns are all addressed.

Reply:
yes, it's suppose to be namespaced